kai_mcp_solution_server/tests/test_multiple_integration.py

```
@@ -353,3 +359,183 @@
+
+        try:
+            self.http_server_process = subprocess.Popen(
+                [
+                    "python",
+                    "-m",
+                    "kai_mcp_solution_server",
+                    "--transport",
+                    "streamable-http",
+                    "--port",
+                    "8087",
+                ],
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)